<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Published Assets Reports</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="selectorLayout" columnstretch="0,1">
      <item row="1" column="1">
       <widget class="QComboBox" name="episodeBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="projectsBox"/>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="sequenceBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sequenceLabel">
        <property name="text">
         <string>Sequence</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="projectLabel">
        <property name="text">
         <string>Project</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Episode</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="shotLabel">
        <property name="text">
         <string>Shot</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="shotBox"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="detailsFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionRecursive"/>
    <addaction name="actionDetails"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionRecursive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Recursive</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionDetails">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Details</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
